packages/cubemap-tiles-adapter/src/CubemapTilesAdapter.ts

```diff
@@ -100,6 +100,7 @@ export class CubemapTilesAdapter extends AbstractAdapter<CubemapTilesPanorama |
             this.loader = new ImageLoader();
             if (this.viewer.config.withCredentials) {
                 this.loader.setWithCredentials(true);
```

Comment:
remove the existing line, it has no effect https://github.com/mrdoob/three.js/blob/dev/src/loaders/ImageLoader.js